src/screen.cpp

```diff
@@ -456,6 +456,12 @@ bool Screen::keyboardCharacterEvent(unsigned int codepoint) {
     return false;
 }
 
+bool Screen::resizeEvent(const Vector2i& size) {
+    if (mResizeCallback)
+        mResizeCallback(size);
```

Comment:
Sorry, one more thing: return ``true`` when there is a callback? (as the event was handled)